include/swift/AST/IRGenOptions.h

```diff
@@ -45,7 +45,10 @@ enum class IRGenOutputKind : unsigned {
   NativeAssembly,
 
   /// Generate an LLVM module, compile it, and assemble into an object file.
-  ObjectFile
+  ObjectFile,
+
+  /// Generate a YAML dump of all fixed size type info in all imported modules.
+  TypeInfo,
```

Comment:
Can you put this after `Module` instead? They're ordered by which LLVM passes to run.

Reply:
Actually, after reading the whole patch I'm not sure you need this at all.